app/jobs/resolution_proofing_job.rb

```diff
@@ -12,9 +12,16 @@ class ResolutionProofingJob < ApplicationJob
     keyword_init: true,
   )
 
-  def perform(result_id:, encrypted_arguments:, trace_id:, should_proof_state_id:,
-              dob_year_only:, user_id: nil, threatmetrix_session_id: nil,
-              request_ip: nil)
+  def perform(
+    result_id:,
+    encrypted_arguments:,
+    trace_id:,
+    should_proof_state_id:,
+    user_id: nil,
+    threatmetrix_session_id: nil,
+    request_ip: nil,
+    dob_year_only: nil, # retired
```

Comment:
```suggestion
    dob_year_only: nil # retired
```
the one place a trailing comma is not allowed in ruby syntax 😭

Reply:
yeah, fixed in a delint commit